web/src/message_list_view.js

```diff
@@ -360,6 +360,10 @@ export class MessageListView {
         const status_message = Boolean(message_container.status_message);
         const edit_history_details = analyze_edit_history(message_container.msg, last_edit_timestr);
 
+        message_container.message_edit_notices_in_left_col = !include_sender && !is_hidden;
+        message_container.message_edit_notices_alongside_sender = include_sender && !status_message;
+        message_container.edited_status_msg = include_sender && status_message;
```

Comment:
Why is this not something like `message_edit_notices_for_status_message`?

Reply:
I didn't modify the variable name for this since for deciding the location of typing notifications we only use the first 2 variables - `.message_edit_notices_in_left_col` and `message_edit_notices_alongside_sender`. So I left the other variables as they are. Should I update it ?

Reply:
I think you're incorrect; it's just in a different template:
```
        {{#if status_message}}                                                                          
            <span class="rendered_markdown status-message">{{rendered_markdown status_message}}</span>  
            {{#if edited_status_msg}}                                                                   
                {{> edited_notice}}                                                                     
            {{/if}}                                                                                     
        {{/if}}                                                                                         
```